composer.json

```diff
@@ -1,68 +1,76 @@
 {
-  "name": "drupal/ui_patterns",
-  "type": "drupal-module",
-  "description": "UI Patterns.",
-  "keywords": ["drupal", "web", "ui"],
-  "license": "GPL-2.0+",
-  "minimum-stability": "dev",
-  "prefer-stable": true,
-  "authors": [
-    {
-      "name": "Nuvole Web",
-      "email": "info@nuvole.org"
+    "name": "drupal/ui_patterns",
+    "type": "drupal-module",
+    "description": "UI Patterns.",
+    "keywords": ["drupal", "web", "ui"],
+    "license": "GPL-2.0+",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "authors": [
+        {
+            "name": "Nuvole Web",
+            "email": "info@nuvole.org"
+        }
+    ],
+    "require-dev": {
+        "composer/installers": "^1.2",
+        "cweagans/composer-patches": "~1.4",
+        "drupal/coffee": "~1",
+        "drupal/config_devel": "~1",
+        "drupal/core-composer-scaffold": "^8.8 || ^9",
+        "drupal/core-dev": "^8.8 || ^9",
+        "drupal/core-recommended": "~8.9.0",
+        "drupal/ds": "~3",
+        "drupal/field_group": "~3",
+        "drupal/page_manager": "*",
+        "drupal/panels": "~4",
+        "drupal/paragraphs": "~1",
+        "drupal/token": "~1",
+        "drush/drush": "~10",
+        "openeuropa/task-runner": "~1.0-beta3",
+        "phpro/grumphp": "~0.14"
+    },
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Drupal\\ui_patterns\\": "./src"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Drupal\\Tests\\ui_patterns\\": "./tests/src"
+        }
+    },
+    "scripts": {
+        "post-install-cmd": "./vendor/bin/run drupal:site-setup",
+        "post-update-cmd": "./vendor/bin/run drupal:site-setup"
+    },
+    "extra": {
+        "composer-exit-on-patch-failure": true,
+        "enable-patching": true,
+        "drupal-scaffold": {
+            "locations": {
+                "web-root": "build/"
+            }
+        },
+        "installer-paths": {
+            "build/core": ["type:drupal-core"],
+            "build/modules/contrib/{$name}": ["type:drupal-module"],
+            "build/profiles/contrib/{$name}": ["type:drupal-profile"],
+            "build/themes/contrib/{$name}": ["type:drupal-theme"]
+        },
+        "patches": {
+            "drupal/coffee": {
+                "Drupal 9 compatibility issue": "https://www.drupal.org/files/issues/2020-06-09/3147664-5.patch"
+            }
+        }
```

Comment:
Let's remove this patch as it will then get applied on all sites. Let's also remove coffee entirely, it's just a convenience dev dependency at the end.